nilearn/glm/contrasts.py

```diff
@@ -262,6 +265,36 @@ def p_value(self, baseline=0.0):
         self.p_value_ = p_values
         return p_values
 
+    def one_minus_pvalue(self, baseline=0.0):
+        """Return a parametric estimate of the 1 - p-value associated with
+        the null hypothesis (H0) where 'contrast equals baseline',
```

Comment:
why "where"?

Reply:
It is a way to phrase what the null hypothesis means. But perhaps, it is not clear. What do you suggest?